pep-0655.rst

```diff
@@ -55,12 +55,25 @@ a mix of both required and potentially-missing keys:
        title: str
        year: NotRequired[int]
 
+This PEP also makes it possible to define TypedDicts in the
+:pep:`alternative functional syntax <589#alternative-syntax>`
+with a mix of required and potentially-missing keys,
+which is not currently possible at all because the alternative syntax does
+not support inheritance:
+
+::
+
+   Actor = TypedDict('Actor', {
+       'name': str,
+       'in': NotRequired[List[str]],  # "in" is a keyword, so needs alternative syntax
+   })
+
 
 Rationale
 =========
 
 One might think it unusual to propose syntax that prioritizes marking
-*required* keys rather than syntax for *potentially-missing* keys, as is
+*required* keys rather than syntax for marking *potentially-missing* keys, as is
```

Comment:
Grammar/semantic issue—currently, due to the use of "prioritizes" between the "than" and the intended target, "syntax", this sentence is naively read as follows:

> One might think it unusual to propose syntax that prioritizes (marking *required* keys) rather than (syntax for marking *potentially-missing* keys), as is

when it appears, from the use of the otherwise semantically out of place "syntax for marking", it is intended to be read as (which may require a few readings to parse correctly):

> One might think it unusual to propose (syntax that prioritizes marking *required* keys) rather than (syntax for marking *potentially-missing* keys), as is

To fix this, I suggest going with the "prioritize" target and eliminating the redundant/repetitive "syntax for marking", which is simpler and more concise,

```suggestion
*required* keys rather than *potentially-missing* keys, as is
```